File: loglr/src/main/java/com/tumblr/loglr/LoginResult.java
package com.tumblr.loglr;

/**
 * Created by guesthouser on 3/1/16.
 */
public class LoginResult {

    /**
     * The Tumblr Key used to make API calls to Tumblr using Jumble client
     */
    private String strTumblrKey;

    /**
     * The Tumblr Secret Key used to make API calls to Tumblr using Jumble client
     */
    private String strTumblrSecreyKey;

    /**
     * @return Return the TumblrKey
     */
    public String getStrTumblrKey() {
        return strTumblrKey;
    }

    /**
     * Sets the Tumblr Key to the object
     * @param strTumblrKey
     */
    void setStrTumblrKey(String strTumblrKey) {
        this.strTumblrKey = strTumblrKey;
    }

    /**
     * @return Return the Tumblr secret key
     */
    public String getStrTumblrSecreyKey() {
        return strTumblrSecreyKey;
    }

    /**
     * @return Return the TumblrKey
     */
    void setStrTumblrSecreyKey(String strTumblrSecreyKey) {
        this.strTumblrSecreyKey = strTumblrSecreyKey;
    }
}

File: Loglr/src/main/java/com/tumblr/loglr/LoginResult.kt
package com.tumblr.loglr

class LoginResult internal constructor() {

    /**
     * The Tumblr Key used to make API calls to Tumblr using Jumble client
     */
    internal var strOAuthToken: String = ""

    /**
     * The Tumblr Secret Key used to make API calls to Tumblr using Jumble client
     */
    internal var strOAuthTokenSecret: String = ""

    /**
     * @return Return the TumblrKey
     */
    fun getOAuthToken(): String = strOAuthToken

    /**
     * Sets the Tumblr Key to the object
     * @param strOAuthToken
     */
    fun setStrOAuthToken(oAuthToken: String) {
        strOAuthToken = oAuthToken
    }

    /**
     * @return Return the Tumblr secret key
     */
    fun getOAuthTokenSecret(): String = strOAuthTokenSecret

    /**
     * @return
     */
    fun setStrOAuthTokenSecret(OAuthTokenSecret: String) {
        strOAuthTokenSecret = OAuthTokenSecret
    }
}